hr_timesheet_default_analytic_account/__manifest__.py

```diff
@@ -1,21 +1,19 @@
-# -*- coding: utf-8 -*-
 # Copyright 2019 Coop IT Easy SCRLfs
 #   - Vincent Van Rossem <vincent@coopiteasy.be>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     "name": "Default analytic accounts upon timesheet creation",
-    "version": "9.0.1.0.0",
+    "version": "12.0.1.0.0",
     "category": "Human Resources",
     "summary": """
     When creating a timesheet, adds for the period analytical
     lines linked to an employee's default analytic accounts.
     """,
     "author": "Coop IT Easy SCRLfs, Odoo Community Association (OCA)",
-    "website": "www.coopiteasy.be",
+    "website": "https://coopiteasy.be",
```

Comment:
-> https://www.coopiteasy.be

Reply:
are you sure ? on `cie-partner-contact` modules it was without `www`, and going to <https://www.coopiteasy.be/> redirects to <https://coopiteasy.be/>. one question i have, though, is whether to add the trailing slash or not.